javascript/todo/src/controls.js

```diff
@@ -0,0 +1,21 @@
+export default class Controls {
+  #titles;
+  #dis;
+  constructor(title, discription) {
```

Comment:
Fix typos : 
titles => title
discription => description

recommended: 
code spell checker extension for vscode 
Extension ID : streetsidesoftware.code-spell-checker